conda_build/post.py

```diff
@@ -320,7 +320,7 @@ def mk_relative(m, f):
     if not is_obj(path):
         return
 
-    if sys.platform.startswith('linux'):
+    if sys.platform.startswith('linux') or ('bsd' in sys.platform):
```

Comment:
That would pretty much be the same dance with `platform.system()`.

    if platform.system() == 'Linux' or ('BSD' in platform.system()):